ee/vellum_ee/workflows/display/editor/types.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
```

Comment:
putting this in `ee/vellum_ee/workflows/display/nodes/types.py` introduced more circ deps bc the `__init__.py` in that package does a full re-export of all the node classes, which it needs to do for Node Display registration. So we started a new package to represent types needed for the sandbox editor experience